nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestWaitNotifyProtocol.java

```diff
@@ -104,8 +105,10 @@ public void testNotifyFirst() throws Exception {
 
         final AtomicCacheEntry<String, String, Long> cacheEntry = cacheEntries.get("signal-id");
 
+        ObjectMapper mapper = new ObjectMapper();
         assertEquals(1, cacheEntry.getRevision().orElse(-1L).longValue());
-        assertEquals("{\"counts\":{\"a\":1},\"attributes\":{},\"releasableCount\":0}", cacheEntry.getValue());
+        assertEquals(mapper.readTree("{\"counts\":{\"a\":1},\"attributes\":{},\"releasableCount\":0}"),
+        mapper.readTree(cacheEntry.getValue()));
```

Comment:
Instead of repeating `mapper.readTree()` for the expected and cached values, it would be helpful to create a new method named `assertValueEquals(String expected, String value)` and then call `mapper.readTree()` in that method.